src/GoogleSheets/packages/auth/create/main.go

```diff
@@ -22,11 +25,30 @@ func HandleRequest(ctx context.Context, event events.APIGatewayProxyRequest) (ev
 			Body:       "Invalid request body",
 		}, nil
 	}
+
+	err = GoogleClient.ConnectSheetsServiceIfNecessary()
+	if err != nil {
+		fmt.Printf("Error connecting to Google Sheets: %v\n", err)
+		return events.APIGatewayProxyResponse{
+			StatusCode: 500,
+			Body:       "Internal server error",
+		}, nil
+	}
+
+	employeeId, err := GetEmployeeId.HandleRequest(signReq.Email)
+	if err != nil {
+		return events.APIGatewayProxyResponse{
+			StatusCode: 500,
+			Body:       fmt.Sprintf("Error getting employeeId: %v", err.Error()),
+		}, nil
+	}
+
+	fmt.Println("Employee ID:", employeeId)
+	clientId := os.Getenv("CLIENT_ID")
```

Comment:
Can we rename the env variable to smth like "COGNITO_CLIENT_ID"? Also, let's do a check to see if it's an empty string. If it's empty, let's return a 500 response saying smth like "No cognito client id found"